airconsole-1.2.1.js

```diff
@@ -245,6 +247,18 @@ AirConsole.prototype.getNickname = function(device_id) {
 };
 
 /**
+ * Returns the device ID of the master controller
+ * @return {number|undefined}
+ */
+AirConsole.prototype.getMasterControllerDeviceId = function() {
```

Comment:
Why not do the following instead.

```
AirConsole.prototype.getControllerDeviceIds = function() {
  var result = [];
  for (var i = AirConsole.SCREEN + 1; i < this.devices.length; ++i) {
    if (this.devices[i]) {
      result.push(i);
    }
  }
  return result;
};
```

It's way more general and used in almost every game.